### PR TITLE
[16.0][MIG] stock_product_qty_by_packaging: Migrate to version 16.0

### PR DESCRIPTION
**Pull request by tuantrantg**

### CODE REVIEW

File: stock_product_qty_by_packaging/readme/CREDITS.rst [hunk 1]
```diff
@@ -1 +1 @@
-The migration of this module from 13.0 to 14.0 was financially supported by Camptocamp
```
Comment (simahawk) [inline stock_product_qty_by_packaging/readme/CREDITS.rst]: this line should stay

File: stock_product_qty_by_packaging/models/stock_move_line.py [hunk 2]
```diff
@@ -10,4 +10,4 @@ class StockMoveLine(models.Model):
     _name = "stock.move.line"
     _inherit = ["stock.move.line", "product.qty_by_packaging.mixin"]
 
-    _qty_by_pkg__qty_field_name = "product_qty"
+    _qty_by_pkg__qty_field_name = "qty_done"
```
Comment (simahawk) [inline stock_product_qty_by_packaging/models/stock_move_line.py]: This is not the same thing. Please check move line fields https://github.com/OCA/OpenUpgrade/blob/de3714a13e5ef57b5abbb2e30ed5080183973cfa/openupgrade_scripts/scripts/stock/16.0.1.1/upgrade_analysis.txt#L55-L60

File: stock_product_qty_by_packaging/tests/test_stock.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                "product_uom": cls.product_a.uom_id.id,
+                "product_uom_qty": 2825,
+                "state": "waiting",
+                "picking_type_id": cls.picking_type.id,
+            }
+        )
+        cls.move._assign_picking()
+        cls.move._action_assign()
+        cls.move_line = cls.move.move_line_ids[0]
+        cls.move_line.qty_done = 1470
```
Comment (simahawk) [inline stock_product_qty_by_packaging/tests/test_stock.py]: this must be fixed w/ the right field, see above

**Review by simahawk — CHANGES_REQUESTED** [review]

File: stock_product_qty_by_packaging/models/stock_move_line.py (hunk 2 above)
Comment (tuantrantg) [reply stock_product_qty_by_packaging/models/stock_move_line.py]: hi @simahawk , you're right, it's my bad, I will replace it by `reserved_qty` in `16.0`

**Comment by tuantrantg** [comment]
hi @simahawk , the PR updated, please review it again, thank you

**Review by simahawk — COMMENTED** [review]
LG, however the build is :red_circle:

**Comment by tuantrantg** [comment]
> LG, however the build is 🔴

hi @simahawk the build is green, please review this PR again, thank you

**Review by sebalix — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

**Pull request merged**